framework/encode/capture_manager.cpp

```diff
@@ -212,6 +211,70 @@ void CommonCaptureManager::DestroyInstance(ApiCaptureManager* api_capture_manage
     }
 }
 
+int32_t CommonCaptureManager::GetPidFromProcessName(const char* process_name)
+{
+    int32_t pid = -1;
+#if defined(__linux__) || defined(__APPLE__)
+    int            id            = 0;
+    DIR*           dir           = nullptr;
+    FILE*          fp            = nullptr;
+    struct dirent* entry         = nullptr;
+    char           filename[256] = { 0 };
+    char           cmdline[256]  = { 0 };
+
+    if (process_name == nullptr)
+    {
+        return pid;
+    }
+    dir = opendir("/proc");
```

Comment:
There's no "/proc" on my macbook.

Reply:
This has been replace with the BSD command, can you verify latest @bradgrantham-lunarg ?